.evergreen/run-aws-lambda-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Needed for cargo-lambda
+pip3 install ziglang
```

Comment:
Are we using Zig for its cross compilation toolchain, i.e., `zig cc`?

Reply:
I believe that's what `cargo-lambda` uses it for, yeah.  The important part from my PoV is it refuses to run if it isn't installed :)